src/model_outputfiles.cpp

```diff
@@ -112,11 +113,17 @@ void Model::createReport(std::string path){
   output_report << vol_block("Van der Waals volume", _data.volumes[0b00000011]);
   output_report << vol_block("Excluded void volume", _data.volumes[0b00000101]);
   output_report << vol_block("Molecular volume", _data.volumes[0b00000011] + _data.volumes[0b00000101], "(vdw + probe inaccessible)");
+  if(!_data.probe_mode && !_data.analyze_unit_cell){
+    output_report << "Warning: the following value includes outside space and is not meaningful\n";

```

Comment:
Agreed. I felt sad for calculating values and never showing them but they are meaningless and serve no purpose so I'll replace the value with "Not applicable (includes unlimited outside space)". I could add "Use two probes mode and/or unit cell analysis to define limits" but that seems too much. The relevant information will be in the user manual.

Reply:
I find the phrasing of "Not applicable (includes unlimited outside space)" too unclear and a bit confusing. I would suggest either shortening is to "Not applicable" or even completely removing the values altogether. What do you think?

Reply:
I don't like to give "not applicable" without further explanation.
We could refer to the user manual and give a more detailed explanation there if you prefer.

Removing the value entirely is doable but as a user, if I wanted to see this value and don't see anything, I would think there is a bug or an incomplete feature.

Reply:
What about "Non-physical value" then?

Reply:
Not a big fan of this proposition:
All values calculated are virtual and not physical to begin with.
The value provided before didn't have a physical meaning but it was given in a form corresponding to physical values, so it was as physical as any other value.

I would still prefer something like:
- "No physical meaning (includes outside space)"
Yet, this is not sufficient for an explanation because probe shell volumes also include outside space (with a clear limitation).
- "Not defined (includes outside space)" 
would be acceptable for me since the words "not defined" may imply the lack of limit.

Any other proposition  ?

Reply:
My issue is more with "outside space". Reading this would immediately trip me up.

"No physical meaning, contains all volume outside the molecule"

Reply:
> "No physical meaning, contains all volume outside the structure"

This is fine for me.

Reply:
Perfect